taxi/forms.py

```diff
@@ -0,0 +1,46 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class BaseLicenseForm(forms.ModelForm):
+    def clean_license_number(self) -> str:
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != 8:
+            raise ValidationError(
+                "License number must contain 8 symbols"
+            )
+        for char in license_number[:3]:

```

Comment:
Simplify to:
`if (not license_number[0:3].isupper() or not license_number[:3].isalpha():`